.travis.yml

```diff
@@ -36,6 +36,7 @@ env:
 
 before_script:
   - mix deps.get
+  - mix credo
```

Comment:
I would have put this in the script phase just above the `mix test` command but I'm not an expert in Travis so let me know what do you think